scalding-dagon/src/main/scala/com/twitter/scalding/dagon/Dag.scala

```diff
@@ -34,7 +34,7 @@ sealed abstract class Dag[N[_]] extends Serializable { self =>
    * the law that for all Expr, the node they
    * evaluate to is unique
    */
-  protected def idToExp: HMap[Id, Expr[N, ?]]
+  protected def idToExp: HMap[Id, Expr[N, *]]
```

Comment:
these type of changes are based on https://github.com/typelevel/kind-projector/pull/172